admin/fleetcommander/fcad.py

```diff
@@ -170,7 +170,10 @@ def _get_smb_connection(self, service="SysVol"):
         creds.set_kerberos_state(MUST_USE_KERBEROS)
         # Create connection
         conn = libsmb.Conn(
-            self._get_server_name(), service, lp=parm, creds=creds, sign=False
+            self._get_server_name(),
+            service,
+            lp=parm,
+            creds=creds,  # sign=False
```

Comment:
Can I know more about this change? Why is required? Any issues?

Reply:
AFAIR the change was done because the libsmb library has changed the signature of that method and the sign attribute was removed, throwing an error of unknown parameter.

Reply:
Moved to a different PR